webui/react/src/components/ClusterMessage.test.tsx

```diff
@@ -0,0 +1,37 @@
+// components/Counter.test.tsx
+import { render, screen } from '@testing-library/react';
+
+import { ClusterMessage } from 'stores/determinedInfo';
+import ClusterMessageBanner from './ClusterMessage';
+
+const setup = (msg?: ClusterMessage) => {
+  render(<ClusterMessageBanner message={msg} />); // render arbitrary components
+};
+
+const msg = 'this is a test msg for use in the ClusterMessageBanner test';
+
+describe('ClusterMessageBanner', () => {
+  it('should have a banner with cluster message text', () => {
+    const time = new Date();
+    const testMsg = { createdTime: time, endTime: time, message: msg, startTime: time };
+
+    setup(testMsg);
+    // make sure these components exist.
+    expect(screen.getByTestId('admin-msg')).toBeInTheDocument();
+    expect(screen.getByTestId('cluster-msg')).toBeInTheDocument();
```

Comment:
instead of using `toBeInTheDocument` in this test case, can we check texts? 
`.toHaveTextContent('text')`